app/code/Magento/Inventory/Model/Reservation/Command/GetListInterface.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\Reservation\Command;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\InventoryApi\Api\Data\ReservationSearchResultsInterface;
+
+/**
+ * Find Reservations by SearchCriteria command (Service Provider Interface - SPI)
+ *
+ * Separate command interface to which Repository proxies initial GetList call, could be considered as SPI - Interfaces
+ * that you should extend and implement to customize current behaviour, but NOT expected to be used (called) in the code
+ * of business logic directly
+ *
+ * @see \Magento\InventoryApi\Api\ReservationRepositoryInterface
+ * @api
+ */
+interface GetListInterface
```

Comment:
don't think we need this interface if we will simplify filtering logic as described above